ESE-FAQ.md

```diff
@@ -132,9 +131,11 @@ Note that Privacy is absent from the examples; values such as Respect and Trust
 * What is in the method engineering backlog?
   * *Answer:* Our work-in-progress and [Future Work](/ESE-FutureWork.md) includes additional application examples, (possibly) showing yet more notations. We also plan to provide more elaborate Markdown templates for ESE and IEEE Std. 7000 artifacts once/when the method content has matured and reached a steady state.
 
-* How can I contribute, for instance by reporting a question missing here?
-  * *Answer:* See [Contributions](/contributing/) folder and readme.
+* How can I contribute, for instance by asking a question missing?
```

Comment:
"by asking a question missing" sounds a bit odd to me, what are we suggesting readers should do?